docs/architecture/tablets.rst

```diff
@@ -0,0 +1,92 @@
+=========================================
+Data Distribution  with Tablets
+=========================================
+
+A ScyllaDB cluster is a group of interconnected nodes - on-premises servers or 
+virtual servers (public cloud instances). The data of the entire cluster has to
+be distributed as evenly as possible across those nodes.
+
+ScyllaDB is designed to ensure a balanced distribution of data when adding new
```

Comment:
Why only scale out?
This is also true for scaling down, adding or removing a data ceneter, or replacing a node